examples/command/portals/ai/amazon_ec2/aws_cli/ai_corp/run.sh

```diff
@@ -64,10 +67,58 @@ run() {
             echo "Downloaded."
 
             npm install express node-llama-cpp
+
+            echo "Start the AI API"
             nohup node api.mjs &>output.log &
+            echo "AI API started"
 EOS
 }
 
+# This function checks if an instance type is available in the currently configured region
+# If not, then the list of all regions supporting that instance type are returned
+check_if_instance_type_available() {
+    instance_type=$1
+    current_region=$(get_configured_region)
```

Comment:
I learnt recently that this is the best way to get region because the it gets it from an api response instead of deciphering it from config and env vars

```
region=$(aws ec2 describe-availability-zones --query 'AvailabilityZones[0].[RegionName]')
```

Reply:
Thanks, I adjusted the script.